src/trio/socket.py

```diff
@@ -67,13 +67,19 @@
     ntohs as ntohs,
 )
 
+# TODO: update python docs to reflect this
```

Comment:
mostly just this that's a blocker

Reply:
Hopefully done with https://github.com/python/typeshed/pull/11073 and https://foss.heptapod.net/pypy/pypy/-/issues/4040.

Reply:
Well I made a typeshed PR thinking "oh well the stdlib docs are the spec, so PyPy is deviating from it and should not have the docs update for it" but then now I realize `sys.implementation.name` isn't a guard typeshed allows :(

And it looks like these functions just never exist on PyPy. Lol.